chainercv/datasets/__init__.py

```diff
@@ -3,6 +3,8 @@
 from chainercv.datasets.camvid.camvid_dataset import camvid_label_names  # NOQA
 from chainercv.datasets.camvid.camvid_dataset import CamVidDataset  # NOQA
 from chainercv.datasets.cityscapes.cityscapes_semantic_segmentation_dataset import CityscapesSemanticSegmentationDataset  # NOQA
+from chainercv.datasets.cityscapes.cityscapes_test_image_dataset import CityscapesTestImageDataset  # NOQA
+from chainercv.datasets.cityscapes.cityscapes_utils import cityscapes_labels  # NOQA
```

Comment:
Can you delete this line? This may confuse users because there is not sufficient description about this object.
We are only exposing `label_names` and `label_colors` directly below `chainercv.datasets`.